src/library-authoring/units/LibraryUnitPage.tsx

```diff
@@ -47,15 +50,14 @@ export const LibraryUnitPage = () => {
     setDefaultTab({
       collection: COLLECTION_INFO_TABS.Details,
       component: COMPONENT_INFO_TABS.Manage,
-      unit: UNIT_INFO_TABS.Manage,
+      container: CONTAINER_INFO_TABS.Manage,
     });
-    setHiddenTabs([COMPONENT_INFO_TABS.Preview, UNIT_INFO_TABS.Preview]);
+    setHiddenTabs([
+      COMPONENT_INFO_TABS.Preview,
+      CONTAINER_INFO_TABS.Preview,
+    ]);
```

Comment:
@pomegranited I think we need to do something similar to hide the Privew tab in the Subsection/Section page. I see that in the [figma designs](https://www.figma.com/design/4wsTUbieYphB2qchIPxm10/%E2%9C%8F%EF%B8%8F-Library---Course-Authoring-v1.0--Open-edX-?node-id=7512-294545&t=fATOrLLqOSRxSNVu-4)

Reply:
Ugh.. I agree this needs doing, but I tried to refactor things while fixing this and ran out of time today.
Will have to finish this up on Monday.

Reply:
Addressed with [294dcca](https://github.com/openedx/frontend-app-authoring/pull/2066/commits/294dccae71a7ad4bcdce89ba0603d19ecc457ce5).